docs/best/actions.md

```diff
@@ -0,0 +1,212 @@
+---
+id: actions
```

Comment:
If I understood it correctly, id could be omitted, in which case file can be used when linking (so the implicit idea is `best/actions`)

Reply:
Removed, and it didn't seem to break anything. Struggling to remember why I felt I had to add them, because it was certainly a PITA to do so! 

Also fixed a couple mixed up docs / titles in the process.